charts/stable/satisfactory/values.yaml

```diff
@@ -0,0 +1,85 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: wolveix/satisfactory-server
+  # -- image tag
+  # @default -- chart.appVersion
+  tag:
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See more environment variables in the [wolverix docker documentation](https://github.com/wolveix/satisfactory-server).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # AUTOPAUSE: true # pause game when no player is connected
+  # AUTOSAVEINTERVAL: 300 # autosave interval in seconds
+  # AUTOSAVENUM: 3 # number of rotating autosave files
+  # AUTOSAVEONDISCONNECT: true # autosave when last player disconnects
+  # CRASHREPORT: true # automatic crash reporting
+  # DEBUG: false # for debugging the server
+  # DISABLESEASONALEVENTS: false # disable the FICSMAS event (you miserable bastard)
+  # MAXPLAYERS: 4 # set the player limit for your server
+  # PGID: 1000 # set the group ID of the user the server will run as
+  # PUID: 1000 # set the user ID of the user the server will run as
+  # SERVERBEACONPORT: 15000 # set the game's beacon port
+  # SERVERGAMEPORT: 7777 # set the game's port
+  # SERVERIP: 0.0.0.0    # set the game's ip (usually not needed)
+  # SERVERQUERYPORT: 15777 # set the game's query port
+  # SKIPUPDATE: false # avoid updating the game on container start/restart
+  # STEAMBETA: false # set experimental game version
```

Comment:
Any particular reason you did not go with the `# --` style comment formatting here?

Reply:
I simply overlooked it! Will fix.

Reply:
Actually, im not sure how the template will go as the actual 'comment' is the second #, not the first (as im commenting out default ENV vars.  Commiting and well see?